hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/SparkFileFormatInternalRowReaderContext.scala

```diff
@@ -51,6 +61,28 @@ class SparkFileFormatInternalRowReaderContext(baseFileReader: PartitionedFile =>
                                      requiredSchema: Schema,
                                      conf: Configuration): ClosableIterator[InternalRow] = {
     val fileInfo = sparkAdapter.getSparkPartitionedFileUtils.createPartitionedFile(partitionValues, filePath, start, length)
-    new CloseableInternalRowIterator(baseFileReader.apply(fileInfo))
+    if (filePath.toString.contains(HoodieLogFile.DELTA_EXTENSION)) {
```

Comment:
Use `FsUtils.isLogFile` instead.

Reply:
`FsUtils.isLogFile` is fixed by considering `inlinefs://` scheme.